http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -262,6 +262,16 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       (labelMap, includeDeleted, _) <- F.fromEither(processListParameters(params))
       excludeStatuses = if (includeDeleted) List.empty else List(RuntimeStatus.Deleted)
       creatorOnly <- F.fromEither(processCreatorOnlyParameter(userInfo.userEmail, params, ctx.traceId))
+
+      // prevent misbehaved user input, labelMap keys and values can not have a single quote
+      // keys and values are inserted directly into a SQL string which could cause sql injection
+      // story #IA-xxx will refactor the SQL string into Slick
+      _ <- F.raiseWhen(
+        labelMap.values.exists(value => value.contains("'")) || labelMap.keys.exists(key => key.contains("'"))
+      )(
+        BadRequestException(s"Invalid query parameter (single quote not allowed)", Some(ctx.traceId))
+      )
```

Comment:
This changes the behavior for runtimes, I think label maps are used in disks and apps as well, should we add similar logic to those services?

Reply:
that might be out of the scope of this particular ticket, but wanted to bring it up